test/headers_self_contained/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Create tests that contain each header seperately. We do not not run these test
```

Comment:
```suggestion
# Create tests that contain each header separately. We do not  run these tests
```

Reply:
`You do not have permission to push to this repository. Refresh and try again.`??!

Reply:
This makes no sense I pushed to it earlier

Reply:
Or was it because it was on a branch associated to a PR